WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/Print Shipping Label/PrintShippingLabelViewController.swift

```diff
@@ -248,24 +316,50 @@ private extension PrintShippingLabelViewController {
     func configureCommonStylesForInfoLinkCell(_ cell: ImageAndTitleAndTextTableViewCell) {
         cell.selectionStyle = .default
     }
+
+    func configurePrintButtonRow(cell: ButtonTableViewCell) {
+        cell.configure(style: .primary,
+                       title: Localization.printButtonTitle,
+                       topSpacing: Constants.buttonVerticalSpacing,
+                       bottomSpacing: Constants.buttonVerticalSpacing) {
+            self.printShippingLabel()
+        }
```

Comment:
nit: It's important here to add `[weak self]` because we don't want a strong reference that can cause a retain cycle 😉 
```suggestion
        cell.configure(style: .primary,
                       title: Localization.printButtonTitle,
                       topSpacing: Constants.buttonVerticalSpacing,
                       bottomSpacing: Constants.buttonVerticalSpacing) { [weak self] in
            self?.printShippingLabel()
        }
```

Reply:
Agh, thanks for catching all of these places where I forgot to add `[weak self]`!